### PR TITLE
update script to work with new qobuz player

### PR DESCRIPTION
**Pull request by Romunix**
Hello,

I fixed the qobuz integration on my PC and as there is no more maintainer, i create an account and i send you the patch.
The patch fix MPRIS2 integration and Multimedia keys.

Issue: tiliado/nuvolaruntime#619
Issue: tiliado/nuvolaruntime#634

### CODE REVIEW

**Review by jiri-janousek — CHANGES_REQUESTED** [review]
Thanks for your PR. Could you run `standard --fix` from [Nuvola ADK](https://github.com/tiliado/nuvolaruntime/wiki/Nuvola-App-Developer-Kit) to fix the coding style? Thanks.

**Comment by Romunix** [comment]
I have installed  Nuvola ADK and i have launched the command. Now i'm ready for maintenance.

**Review by jiri-janousek — CHANGES_REQUESTED** [review]
Code looks good :-)

* Could you add yourself (`Copyright 2020 Name <email>`) to the list of copyright holders in `README.md` and at the top of `integrate.js`?
* Could you update `CHANGELOG.md`?

**Comment by Romunix** [comment]
Files updated ;-)

**Review by jiri-janousek — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ Qobuz App Change Log
 
 * Quobuz is unsupported until a new maintainer is found. Issue: tiliado/nuvolaruntime#634
 
+1.3 - October 4th, 2020
+------------------------
```
Comment (jiri-janousek) [inline CHANGELOG.md:7]: We write the release number and date when a release is made. I've changed it to `unreleased`.

**Comment by jiri-janousek** [comment]
Merged and fresh flatpak packages published.

**Pull request merged**